### PR TITLE
should be using method index instead of service index for method's pr…

### PR DESCRIPTION
**Pull request by coolyrat**
tested on my [use case ](https://github.com/coolyrat/grpc-gateway-merge-test)

### CODE REVIEW

**Comment by coolyrat** [comment]
@googlebot I signed it!

**Comment by achew22** [comment]
Can you regenerate the examples so that we can see how this change affects things? Thanks!

**Comment by johanbrandhorst** [comment]
See CONTRIBUTING.md for instructions :)

**Comment by coolyrat** [comment]
I'm not sure what's wrong, could you guys help and have a peek

**Comment by johanbrandhorst** [comment]
@googlebot I consent.

File: examples/internal/proto/examplepb/unannotated_echo_service.swagger.json [hunk 1]
```diff
@@ -47,8 +47,6 @@
     },
     "/v1/example/echo/{id}/{num}": {
       "get": {
-        "summary": "Echo method receives a simple message and returns it.",
-        "description": "The message posted as the id parameter will also be\nreturned.",
```
Comment (johanbrandhorst) [inline examples/internal/proto/examplepb/unannotated_echo_service.swagger.json]: This change is incorrect, so we need to figure out how we can fix your case and still keep this.

**Comment by coolyrat** [comment]
@johanbrandhorst i've tried every thing i could to make the ci happy. But there's nothing I could do about it. Need your help to regenerate.

File: examples/internal/proto/examplepb/a_bit_of_everything.swagger.json [hunk 2]
```diff
@@ -2037,6 +2037,8 @@
     },
     "/v2/example/empty": {
       "get": {
+        "summary": "Create a new ABitOfEverything",
+        "description": "This API creates a new ABitOfEverything",
```
Comment (johanbrandhorst) [inline examples/internal/proto/examplepb/a_bit_of_everything.swagger.json]: This comment is not correct either. There should not be a comment here. See https://github.com/grpc-ecosystem/grpc-gateway/blob/master/examples/internal/proto/examplepb/a_bit_of_everything.proto#L541-L547

**Comment by johanbrandhorst** [comment]
Stale

**Pull request closed without merging**